scene/gui/text_edit.cpp

```diff
@@ -2597,6 +2597,13 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 				}
 				FALLTHROUGH;
 			}
+#ifdef APPLE_STYLE_KEYS
+			case KEY_B:
+				if (!k->get_control()) {
+					scancode_handled = false;
+					break;
+				}
```

Comment:
These should have `FALLTHROUGH;` to tell the compiler about the intentional fallthrough, as done for other cases.

The commit log should also be updated to be more explicit, "Update text_edit.cpp" is not.

Reply:
Updated.